releasenotes/notes/context-d43f1c79c78cb923.yaml

```
@@ -0,0 +1,23 @@
+---
+prelude: >
+    Major breaking changes to context management. See the upgrade section for
+    the specifics. Note that only advanced users of the library should be
+    affected by the changes.
+features:
+  - |
+    `tracer.start_span()` now accepts an `activate` argument (default False) to
+    allow manual context management."
+upgrade:
+  - |
+    `ddtrace.span.Span`
+    - `Span.context` will now return a `Context`
+  - |
+    `ddtrace.tracer.Tracer`
+    - "`tracer.get_call_context()` will now return None if there is no active trace."
```

```suggestion
    - ``tracer.get_call_context()`` will now return ``None`` if there is no active trace.
```
not sure you need the `"`?

---

It was necessary because YAML complains if a line starts with ` 😕

---

Oh I see yeah because of the `|` this should be necessary anymore :+1: